flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -266,6 +313,81 @@ def process(self, element, **kwargs):
         yield operation.item
 
 
+@experimental()
+class MaskDetectedDetails(PTransform):
+
+    def __init__(
+            self,
+            project=None,
+            location="global",
+            template_name=None,
+            deidentification_config=None,
+            timeout=None):
+
+        self.config = {}
+        self.project = project
+        self.timeout = timeout
+        self.location = location
+
+        if template_name is not None:
+            self.config['deidentify_template_name'] = template_name
+        else:
+            self.config['deidentify_config'] = deidentification_config
+
+    def expand(self, pcoll):
+        if self.project is None:
+            self.project = pcoll.pipeline.options.view_as(
+                GoogleCloudOptions).project
+        if self.project is None:
+            raise ValueError(
+                'GCP project name needs to be specified '
+                'in "project" pipeline option')
```

Comment:
is the pipeline option really `dlp_project` ?  just check to match what the user sees in the options matches the context in the error message.

also, add a period (.) at the end of the sentence.